### PR TITLE
Fix [[graphite]] config section

### PR DESCRIPTION
**Pull request by jackzampolin**

### CODE REVIEW

**Comment by jackzampolin** [comment]
@jwilder :eyes: This reverts the default configuration file to the `template` style

File: etc/config.sample.toml [hunk 1]
```diff
@@ -232,36 +232,32 @@ reporting-disabled = false
...
-  ##         "device": "cpu0"
-  ##     }
-  ## }
-  # name-schema = "type.host.measurement.device"
-
-  ## If set to true, when the input metric name has more fields than `name-schema` specified,
-  ## the extra fields will be ignored.
-  ## Otherwise an error will be logged and the metric rejected.
-  # ignore-unnamed = true
+  ### If matching multiple measurement files, this string will be used to join the matched values.
```
Comment (jwilder) [inline etc/config.sample.toml]: `files` isn't quite the right name.  Maybe rework along the lines of how it's described here: https://github.com/influxdata/influxdb/tree/master/services/graphite#multiple-measurement-matching?

File: etc/config.sample.toml [hunk 2]
```diff
@@ -232,36 +232,32 @@ reporting-disabled = false
...
+  # separator = "."
+
+  ### Default tags that will be added to all metrics.  These can be overridden at the template level
+  ### or by tags extracted from metric
+  # tags = ["region=us-east", "zone=1c"]
+
+  ### Each template line requires a template pattern.  It can have an optional
+  ### filter before the template and separated by spaces.  It can also have optional extra
+  ### tags following the template.  Multiple tags should be separated by commas and no spaces
+  ### similar to the line protocol format.  The can be only one default template.
```
Comment (jwilder) [inline etc/config.sample.toml]: There can be...

**Comment by jackzampolin** [comment]
@jwilder

**Comment by jwilder** [comment]
:+1:

**Pull request merged**